### DISCUSSION TITLE
Welcome to snappymail Discussions!

### DISCUSSION THREAD

**Discussion by the-djmaze** [the-djmaze/snappymail#556] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by webagroprom** [comment]
```
Release 2.19.0.1 and 2.19.01 not possible and 2.19.0-1 gets marked as unstable.
So i've made it 2.19.1

You must manually update extension/plugin to 2.5!

It includes:

PharData workaround (just try it)
Plugin has basic Nextcloud folder selector (no sub-tree yet) to save attachments/email
Fix for Content-Security-Policy
Some other cleanups and improvements

```

**After installing snappymail through the admin panel, I see that the plugin is installed 2.5**

![1666173271122](https://user-images.githubusercontent.com/30408997/196660516-07d3b8da-5ca5-4698-aef4-108eb5457c76.jpg)

1. Do I need to manually update the plugin in this case?
2. How to update the plugin manually?
3. Where can I see that the plugin is working correctly and take advantage of its features?

**When creating a message, I do not see the buttons for this plugin.**
Where can I find the nextcloud 2.5 plugin button for the nextcloud attachment to work?

![1666175792877](https://user-images.githubusercontent.com/30408997/196668464-6b4c0df6-e76f-4500-b7ac-b54f05a832fe.jpg)

**Reply by the-djmaze** [reply]
1. Do I need to manually update the plugin in this case?

No, it should work fine then.

2. How to update the plugin manually?

There at Admin -> Extensions (when they are available)

3. Where can I see that the plugin is working correctly and take advantage of its features?

Open/view a message:
1. Hit the message menu button and at the bottom there's a "Save .eml in Nextcloud"
2. Click the attachments cog wheel (when there are attachments) and click `Save to Nextcloud`

Currently it's not possible to send Nextcloud files.

**Comment by webagroprom** [comment]
1. Is this app going to be supported?
2. What features are planned for development in the first place?
3. **Is there a feature planned:** 

- Аdding files from nextcloud
- Adding Files Creating a Temporary / Permanent Link to an Attachment

**Comment by the-djmaze** [comment]
1. yes
2. standalone version and then the Nextcloud version will have them too
3. Only when someone (or i) have the time
